python/mathsys/dev/builder/code.py

```
@@ -0,0 +1,75 @@
+        try: 
+            subprocess.run(
+                self.command(target, filename),
+                cwd = os.path.dirname(os.path.abspath(__file__)),
+                env = environment,
+                capture_output = False,
+                text = True,
+                check = True
+            )
+            with open(filename, "rb") as file: binary = file.read()
+            return binary
+        except: raise
```

Ruff (0.14.6)

Remove exception handler; error is immediately re-raised

(TRY203)